### DISCUSSION TITLE
Slider ends aren't grouped together with slider ticks in the results screen

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#26507] [Q&A] [answered]
In the results screen, slider ends have the same light blue color as spinner bonus and spinner spin, when it should probably be blue just like slider ticks. Also, slider ends are separated from slider ticks when they should be next to each other.

How it looks now:

![image](https://github.com/ppy/osu/assets/74088315/3c60284b-d37d-4c26-958e-2e4391131ba9)

How it used to look:

![image](https://github.com/ppy/osu/assets/74088315/9f14fb57-54d0-4f57-8242-62764e8a3a94)

**Comment by peppy** [comment]
I meant to fix this because it was bugging me, but forgot.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/26522
